ptop.py

```diff
@@ -60,7 +60,9 @@ def ascii_progress_bar(percentage, bar_length=50):
                 lines.extend(new_lines)
 
             clear_console()
-            print(ascii_progress_bar(progress(lines, tj)))
+            if tj > 0: 
```

Comment:
```suggestion
            if tj > 0:
```

Pls remove the trailing whitespace.